src/main/java/usf/sdlc/form/StockHistoryForm.java

```diff
@@ -0,0 +1,44 @@
+package usf.sdlc.form;
+
+public class StockHistoryForm {
```

Comment:
Will you use this class or not, because i saw you comment out all

Reply:
No, I will not be using this class